pinot-controller/src/test/java/org/apache/pinot/controller/api/extraresources/PinotDummyExtraRestletResource.java

```diff
@@ -26,13 +26,17 @@
 import javax.ws.rs.Path;
 import javax.ws.rs.Produces;
 import javax.ws.rs.core.MediaType;
+import org.apache.pinot.core.auth.Actions;
+import org.apache.pinot.core.auth.Authorize;
+import org.apache.pinot.core.auth.TargetType;
 
 
 @Api(tags = "testExtra")
 @Path("/testExtra")
 public class PinotDummyExtraRestletResource {
 
   @GET
+  @Authorize(targetType = TargetType.CLUSTER, action = Actions.Cluster.DUMMY)
```

Comment:
This is test only API (not visible), do not add it

Reply:
If neither Authorize nor ManualAuthorization is present, then the AUTH call will go to defaultAuthorization and that may not be allowed. If that is the case, no user will be able to use this test API. I suggest to keep the annotation and change the action to Test or Debug, what do you say?

Reply:
Same as the `echo`, this resource file is in the test package, so not included in the production code